uc-crux-llvm/src/UCCrux/LLVM/Config/FromEnv.hs

```diff
@@ -73,7 +104,35 @@ processUCCruxLLVMOptions (initCOpts, initUCOpts) =
             },
           ucLLVMOptions initUCOpts
         )
-    pure (appCtx, finalCOpts, initUCOpts {ucLLVMOptions = finalLLOpts})
+    pure
+      ( appCtx
+      , finalCOpts
+      , Config.TopLevelConfig
+          { Config.ucLLVMOptions = finalLLOpts,
+            Config.runConfig =
+              case entries of
+                Just ents -> Config.RunOn ents
+                Nothing ->
+                  if doExplore initUCOpts
+                  then
+                    Config.Explore
+                      (ExConfig.ExploreConfig
+                        { ExConfig.exploreAgain = reExplore initUCOpts,
+                          ExConfig.exploreBudget = exploreBudget initUCOpts,
+                          ExConfig.exploreTimeout = exploreTimeout initUCOpts,
+                          ExConfig.exploreParallel = exploreParallel initUCOpts,
+                          ExConfig.exploreSkipFunctions = skipFunctions initUCOpts
+                        })
+                  else
+                    Config.CrashEquivalence
+                      (EqConfig.EquivalenceConfig
+                        { EqConfig.equivStrict =
+                            crashEquivalenceStrict initUCOpts,
+                          EqConfig.equivModule = crashEquivalence initUCOpts,
+                          EqConfig.equivEntryPoints = entryPoints initUCOpts
+                        })
+          }
```

Comment:
Can we extract this definition (the top level config) out into a let binding or something similar to make it more visually obvious what the return triple is?

Reply:
Good idea, done!